### PR TITLE
Slides session 1

### PR DESCRIPTION
**Pull request by beatrizmilz**
Hi @3mmaRand ! 
Here are some small suggestions I made while reading the slides.
Fell free to tell me what suggestions do you want me to keep or remove from the PR, so I can adapt it!
Each commit message talks about why I'm making the suggestion.

Thanks!!

Ps:
This slide seems with no content except the title (but the slide before also had the title, so maybe you thought about adding something here?):

https://rstudio-conf-2022.github.io/build-tidy-tools/materials/day-1-session-1-introduction.html#/why-make-a-package-1

image

### CODE REVIEW

**Review by 3mmaRand — APPROVED** [review]
great, thanks

**Review by 3mmaRand — CHANGES_REQUESTED** [review]
Please could you remove this one @beatrizmilz 
You may need to resize you wondow but it should fit and I want consistency with the other slides

**Review by 3mmaRand — APPROVED** [review]
great, thanks you

**Review by 3mmaRand — APPROVED** [review]

File: materials/day-1-session-1-introduction.qmd [hunk 1]
```diff
@@ -1122,7 +1122,7 @@ Read more in [Choose a Licence](https://choosealicense.com/licenses/).
 ::: small
 -   It is useful to make a package
     -   it is fairly easy with `devtools`
-    -   it will help you work more reproducibly
```
[the remark this answers is not in the document]
Comment (beatrizmilz) [reply materials/day-1-session-1-introduction.qmd]: I did not knew that! :O thanks! I thought that was a typo!

**Comment by beatrizmilz** [comment]
> Please could you remove this one @beatrizmilz You may need to resize you wondow but it should fit and I want consistency with the other slides

Emma, GitHub is not showing what this change is about! Is the changes in "# Where packages come from and live?"? thanks!

**Comment by 3mmaRand** [comment]
> Emma, GitHub is not showing what this change is about! Is the changes in "# Where packages come from and live?"? thanks!

Sorted, don't worry. it was one of the package states slides. Sorry I didn't mark properly

**Pull request merged**